### DISCUSSION TITLE
Success! 🎉

### DISCUSSION THREAD

**Discussion by jaredmoody** [plecong/homebridge-dkncloudna#4] [Show and tell]
Well that was pretty epic turn-around time!

I got the plugin installed, configured and connected to my mini split:

![IMG_04F566952B85-1](https://user-images.githubusercontent.com/27690/180632300-668da681-23b1-4a6a-aa96-a0f4dc3eddd0.jpeg)

Thanks for this plugin!

**Comment by dmthepm** [comment]
Same here. Grateful for this plugin!
